api/app.rb

```diff
@@ -210,6 +210,8 @@ def validate!
 # The two apps are mounted together in rack. This app is mounted under
 # /:version/render
 class RenderApp < Sinatra::Base
+  helpers Sinatra::Cookies
```

Comment:
I don't understand why `Sinatra::Cookies` needs to be included.  It provides only a single method `cookies` which isn't used in the render app.  The cookies are accessed via `request.cookies`.  The inconsistency between how the render app and the non-render app access the cookies is something worth fixing.

However, I'm concerned about why you were getting a 500 error here and I'm not.  Do you have reproduction steps for getting the 500 error?

Reply:
Ah my mistake, I was having issues with cookies on a different branch. Just assumed it was missing the cookie library as it is used higher up in the file.

However this library will be required for future work. So removing it will create conflicts